conjure-java-core/src/integrationInput/java/com/palantir/product/ServerEndpointSpecificTwoErrors.java

```diff
@@ -0,0 +1,28 @@
+package com.palantir.product;
+
+import com.palantir.conjure.java.undertow.lib.CheckedServiceException;
+import javax.annotation.Nullable;
+import javax.annotation.processing.Generated;
+
+@Generated("com.palantir.conjure.java.types.CheckedErrorGenerator")
+public final class ServerEndpointSpecificTwoErrors {
+    private ServerEndpointSpecificTwoErrors() {}
+
+    public static DifferentNamespace differentNamespace() {
```

Comment:
I forget what we landed on when we discussed this live, but I'm opting to not add the `throwIf` variant. It sounds like that is some syntactic sugar that devs can define in their projects if they need.

Reply:
hm, I actually see it's used a bunch. Including the variant will make the migration easier.